.chloggen/adopt_weaver_ntp.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: ntpreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add a weaver model to ntp receiver to generate its metadata.yaml
```

Comment:
Not sure if this deserves a changelog, the end result won't affect the Collector end-user, right?

Reply:
I'll remove it...